### PR TITLE
get rid of references to hpcugent after move to github.com/easybuilders

### PR DESCRIPTION
**Pull request by boegel**
last changes (hopefully) related to the migration to github.com/easybuilders

### CODE REVIEW

File: RELEASE_NOTES [hunk 1]
```diff
@@ -635,7 +635,7 @@ feature + bugfix release
 - add support for using external modules as dependencies, and to provide metadata for external modules (#1230, #1265, #1267)
     - see http://easybuild.readthedocs.org/en/latest/Using_external_modules.html
 - add experimental support for Cray toolchains on top of PrgEnv modules: CrayGNU, CrayIntel, CrayCCE (#1234, #1268)
-    - see https://github.com/hpcugent/easybuild/wiki/EasyBuild-on-Cray for more information
+    - see http://easybuild.readthedocs.io/en/latest/Cray-support.html for more information
```
Comment (damianam) [inline RELEASE_NOTES]: Not too important, but why drop https?

File: RELEASE_NOTES [hunk 2]
```diff
@@ -905,7 +905,7 @@ feature + bugfix release
     - add --upload-test-report command line option for uploading a detailed test report to GitHub as a gist
         - this requires specifying a GitHub username for which a GitHub token is available, using --github-user
         - with --dump-test-report, the test report can simply be dumped to file rather than being uploaded to GitHub
-        - see also https://github.com/hpcugent/easybuild/wiki/Review-process-for-contributions#testing-result
+        - see also http://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#uploading-test-reports-upload-test-report
```
Comment (damianam) [inline RELEASE_NOTES]: Same as previous comment

File: RELEASE_NOTES [hunk 3]
```diff
@@ -1200,7 +1200,7 @@ v1.3.0 (April 1st 2013)
 -----------------------
 
 feature + bugfix release
-- added script to bootstrap EasyBuild with EasyBuild, see https://github.com/hpcugent/easybuild/wiki/Bootstrapping-EasyBuild (#531)
+- added script to bootstrap EasyBuild with EasyBuild, see http://easybuild.readthedocs.io/en/latest/Installation.html#bootstrapping-easybuild (#531)
```
Comment (damianam) [inline RELEASE_NOTES]: Ditto

File: README.rst [hunk 4]
```diff
@@ -14,36 +14,36 @@ The EasyBuild documentation is available at http://easybuild.readthedocs.org/.
...
   * a collection of example easyconfig files that specify which software to build,
     and using which build options; these easyconfigs will be well tested
     with the latest compatible versions of the easybuild-framework and easybuild-easyblocks packages
-  * GitHub repository: http://github.com/hpcugent/easybuild-easyconfigs
+  * GitHub repository: https://github.com/easybuilders/easybuild-easyconfigs
   * PyPi: https://pypi.python.org/pypi/easybuild-easyconfigs
 
 The code in the ``vsc`` directory originally comes from the *vsc-base* package
-(https://github.com/hpcugent/vsc-base).
+(https://github.com/easybuilders/vsc-base).
```
Comment (damianam) [inline README.rst]: `vsc-base` hasn't been migrated

File: easybuild/tools/options.py [hunk 5]
```diff
@@ -100,9 +100,9 @@ def terminal_supports_colors(stream):
             return False
 
 # monkey patch shell_quote in vsc.utils.generaloption, used by generate_cmd_line,
-# to fix known issue, cfr. https://github.com/hpcugent/vsc-base/issues/152;
-# inspired by https://github.com/hpcugent/vsc-base/pull/151
-# this fixes https://github.com/hpcugent/easybuild-framework/issues/1438
+# to fix known issue, cfr. https://github.com/easybuilders/vsc-base/issues/152;
```
Comment (damianam) [inline easybuild/tools/options.py]: `vsc-base` hasn't been migrated

File: easybuild/tools/options.py [hunk 6]
```diff
@@ -100,9 +100,9 @@ def terminal_supports_colors(stream):
...
+# inspired by https://github.com/easybuilders/vsc-base/pull/151
```
Comment (damianam) [inline easybuild/tools/options.py]: The same

File: test/framework/easyconfig.py [hunk 7]
```diff
@@ -313,8 +313,8 @@ def test_exts_list(self):
             '       "source_urls": [("http://example.com", "suffix")],'
             '       "patches": ["toy-0.0.eb"],',  # dummy patch to avoid downloading fail
             '       "checksums": [',
-            '           "9e9485921c6afe15f62aedfead2c8f6e",',  # MD5 checksum for source (gzip-1.4.eb)
-            '           "8ebc2c32692be9ee61eadc5d650cd288",',  # MD5 checksum for patch (toy-0.0.eb)
+            '           "2dc55aee0346ad5e4f0c2e46d554a92e",',  # MD5 checksum for source (gzip-1.4.eb)
+            '           "c8308888dca8bc2ea62efb22be445414",',  # MD5 checksum for patch (toy-0.0.eb)
```
Comment (damianam) [inline test/framework/easyconfig.py]: Since we are kind of migrating to sha256, maybe change the hash here?

**Comment by damianam** [comment]
@boegel looks good to me, but being so many files I think it makes sense to ask for another pair of eyes to take a look, chances are that I overlooked something.

File: test/framework/github.py [hunk 8]
```diff
@@ -55,9 +55,8 @@
 
 # test account, for which a token may be available
 GITHUB_TEST_ACCOUNT = 'easybuild_test'
-# the user who's repo to test
+# the user & repo to use in this test (https://github.com/hpcugent/testrepository)
```
Comment (wpoely86) [inline test/framework/github.py]: move this one too?

Comment (boegel) [inline test/framework/github.py]: Hmm, maybe I could... Not sure it's worth the trouble though?

**Pull request merged**